<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Annotator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="annotator.qrc">
    <normaloff>:/images/accessories-text-editor.png</normaloff>:/images/accessories-text-editor.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="edellinenBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>edellinen (c)</string>
    </property>
    <property name="icon">
     <iconset resource="annotator.qrc">
      <normaloff>:/images/arrow-left.png</normaloff>:/images/arrow-left.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="seuraavaBtn">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>50</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>seuraava (v)</string>
    </property>
    <property name="icon">
     <iconset resource="annotator.qrc">
      <normaloff>:/images/arrow-right.png</normaloff>:/images/arrow-right.png</iconset>
    </property>
   </widget>
   <widget class="QRadioButton" name="hAutoBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Henkilöauto (&amp;1)</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="pAutoBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pakettiauto (&amp;2)</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="kAutoBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Kuorma-auto (3)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nnforgeBtn">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>30</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>nnForge</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="annotator.qrc">
      <normaloff>:/images/justnn.gif</normaloff>:/images/justnn.gif</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>580</y>
      <width>591</width>
      <height>161</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>590</y>
      <width>63</width>
      <height>23</height>
     </rect>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>590</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Annotoitu</string>
    </property>
   </widget>
   <widget class="QLabel" name="imBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>731</width>
      <height>441</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imgName">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>550</y>
      <width>581</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ignoreBtn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>hylkää (n)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1084</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="annotator.qrc"/>
 </resources>
 <connections/>
</ui>
